presto-orc/src/main/java/com/facebook/presto/orc/metadata/DwrfMetadataReader.java

```diff
@@ -159,6 +163,15 @@ private static RowGroupIndex toRowGroupIndex(OrcProto.RowIndexEntry rowIndexEntr
         return ImmutableList.copyOf(Iterables.transform(columnStatistics, statistics -> toColumnStatistics(statistics, isRowGroup)));
     }
 
+    private Map<String, Slice> toUserMetadata(List<OrcProto.UserMetadataItem> metadataList)
```

Comment:
Yeah - I started with `String` as a value but changed it to `Slice` later as this is a bit more versatile. And also with `String` there was bunch of ugly `Stiring`<->`byte[]`<->`ByteBuffer` conversions needed